### PR TITLE
feat: create gitpod version of quickstart

### PR DESCRIPTION
**Pull request by leeederek**
- port over the quickstart main.star file 
- update the readme
- add go and ts tests (from https://github.com/kurtosis-tech/awesome-kurtosis/tree/main/quickstart)

### CODE REVIEW

File: go-test/go.mod [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+	github.com/ulikunitz/xz v0.5.10 // indirect
+	github.com/xi2/xz v0.0.0-20171230120015-48954b6210f8 // indirect
+	golang.org/x/net v0.7.0 // indirect
+	golang.org/x/sys v0.5.0 // indirect
+	golang.org/x/text v0.7.0 // indirect
+	google.golang.org/genproto v0.0.0-20200526211855-cb27e3aa2013 // indirect
+	google.golang.org/grpc v1.38.0 // indirect
+	google.golang.org/protobuf v1.27.1 // indirect
+	gopkg.in/yaml.v3 v3.0.0-20200313102051-9f266ea9e77c // indirect
+)
```
Comment (h4ck3rk3y) [inline go-test/go.mod]: new line

File: README.md [hunk 2]
```diff
@@ -1,5 +1,28 @@
...
+
+The `main.star` file in this directory contains the build steps for a composable and portable multi-container test environment. Specifically, Kurtosis will use those instructions to:
+- [Start a containerized Postgres database in Kurtosis][run-postgres],
+- [Seed your database with test data using task sequencing][add-data],
+- [Connect an API server to your database using dynamic service dependencies][add-an-api], and
+- [Parameterize your application setup in order to automate loading data into your API][insert-data]
+
+Feel free, at any time, to refer to the [conclusion][conclusion] and the other review sections to get an explanation of the concepts and capabilities of Kurtosis.
+
+### To run tests against your test environment:
```
Comment (h4ck3rk3y) [inline README.md:18]: might have to add language support for golang in the Gitpod but we can do that after we merge this too


broadly speaking - Do we want all of this here? Can it just be Starlark?

Comment (leeederek) [reply README.md:18]: We've heard from users that _if_ we are claiming to be a tool for "test environments", then we should very explicitly show them how to run a test. I think that feedback makes a lot of sense and we should leave this here. 

+1 to adding language support for Golang in Gitpod. Can you help me with that?

File: go-test/go.mod (hunk 1 above)
Comment (leeederek) [inline go-test/go.mod]: Does this work?
```suggestion
)

```

File: README.md (hunk 2 above)
Comment (h4ck3rk3y) [reply README.md:18]: Merging this and fixing this without any guardrails on.

**Review by h4ck3rk3y — APPROVED** [review]

**Pull request merged**